documentation/docs/decentralized_identifiers/update.mdx

```diff
@@ -12,19 +12,382 @@ keywords:
 ---
 import account_manipulate_did_rs from '!!raw-loader!../../../examples/account/manipulate_did.rs';
 import CodeSnippet from '../../src/components/CodeSnippetComponent'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
 
 
-TODO: Explain how DID Documents can be manipulated and how updates should be published.
-
 ## Example
 
 This example shows how you can add more to an existing DID Document. The two main things you can add are Verification Methods and Services. A verification method adds public keys, which can be used to digitally sign things like an identity. The services provide metadata around the identity via URIs. These can be URLs, but also emails or IOTA indices.
```

Comment:
The explanation should after the first sentence should be above the Example section as an introduction. Please add links to W3C sections for verification methods and services.